components/prism-webidl.js

```diff
@@ -0,0 +1,12 @@
+Prism.languages.webidl = Prism.languages.extend('c', {
+	'keyword': /\b(any|attribute|boolean|byte|ByteString|callback|const|deleter|dictionary|DOMString|double|enum|float|FrozenArray|getter|identifier|implements|Infinity|inherit|integer|interface|iterable|legacycaller|long|maplike|namespace|NaN|null|object|octet|optional|or|other|partial|readonly|required|sequence|serializer|setlike|setter|short|static|string|stringifier|typedef|unrestricted|unsigned|USVString|void)\b/,
+	'boolean': /\b(true|false)\b/,
+	'operator': /-|:|=|\?|\*|\//
+});
+
+Prism.languages.insertBefore('webidl', 'keyword', {
+	'class-name': {
+		pattern: /(class\s+)[a-z0-9_]+/i,
+		lookbehind: true
+	}
+});
```

Comment:
@LeaVerou The `c` language actually deletes the `class-name` property from `clike`, so if you inherit from `c` and you want to put `class-name` at a certain position, you have to use `insertBefore()`. However if you inherit from `clike`, you can directly add `class-name` to the language.